.github/workflows/validate_pr.yaml

```diff
@@ -0,0 +1,32 @@
+name: Validate_PR
```

Comment:
Let's just call it "Validate PR" to be consistent with the naming conventions of our other repositories.